src/inventory_ui.h

```diff
@@ -0,0 +1,341 @@
+#ifndef INVENTORY_UI_H
+#define INVENTORY_UI_H
+
+#include <memory>
+
+#include "cursesdef.h"
+#include "input.h"
+#include "item_location.h"
+#include "item.h"
```

Comment:
Will do my best. It's not a significant issue here since the header is included only in `game.cpp` which already contains those headers, though.